superset/views/database/views.py

```diff
@@ -91,11 +91,14 @@ def form_post(self, form):
             return redirect("/csvtodatabaseview/form")
 
         csv_file = form.csv_file.data
-        form.csv_file.data.filename = secure_filename(form.csv_file.data.filename)
+
         csv_filename = form.csv_file.data.filename
-        path = os.path.join(config["UPLOAD_FOLDER"], csv_filename)
+        extension = os.path.splitext(csv_filename)[1].lower()
```

Comment:
Probably best if we don't deal with extensions at all.

Reply:
Filename extension is checked on db_engine_spec, so I'm preserving it

Reply:
Make sure this works with no file extension

Reply:
Yes, tested and it's safe